### PR TITLE
Release 0.12.20

### PR DESCRIPTION
**Pull request by martinRenou**
Signed-off-by: martinRenou <martin.renou@gmail.com>

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by martinRenou** [comment]
@conda-forge-admin, please rerender

**Comment by martinRenou** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,21 +17,29 @@ build:
 
 requirements:
   host:
-    - python
+    - python  >=3.6
     - setuptools
     - jupyter
+    - jupyter-packaging
```
Comment (jtpio) [inline recipe/meta.yaml]: Maybe with this lower bound to ensure fixes from the latest releases?

```suggestion
    - jupyter-packaging >=0.7.9
```

**Comment by martinRenou** [comment]
@jtpio thanks for the review!

**Comment by jtpio** [comment]
Looks like all checks are passing :+1:

**Pull request merged**